composer.json

```diff
@@ -16,6 +16,6 @@
         }
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.8"
+        "phpunit/phpunit": "^4.8.10|^5.0"
```

Comment:
Arguably, fixing the lowest version and supporting PHPUnit 5 *could* be two separate PRs, but I'll leave this up to you :+1: